cmd/buildtools/openebs.go

```diff
@@ -78,3 +78,115 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		return nil
 	},
 }
+
+type addonComponent struct {
```

Comment:
can this be moved to a separate file as it'll be used by other addons? maybe `addon.go`?